{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -22,9 +22,16 @@
 import edx_theme
 
 import django
+from django.conf import settings
 from django.utils import six
 
+REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(REPO_ROOT)
+
+import {{ cookiecutter.app_name }}  # pylint: disable=wrong-import-position
```

Comment:
@jmbowman as much as I don't like copy-pasting *functions*, that's seems like the lesser evil here.